rcl/src/rcl/security_directory.c

```diff
@@ -247,12 +247,12 @@ char * rcl_get_secure_root(
     // Check node_secure_root is not NULL before checking directory
     if (NULL == node_secure_root) {
       RCL_SET_ERROR_MSG_WITH_FORMAT_STRING(
-        "SECURITY ERROR: unable to find a folder matching the node name in %s%s."
+        "SECURITY ERROR: unable to find a folder matching the node name '%s' in '%s%s'.\n"
```

Comment:
Sorry for another comment: how about just adding a trailing space instead of a newline? That keeps the lookup strategy on the same line as the error message.

Reply:
Yeah, I originally also printed the node namespace and name on a different line, hence the newline here.
Now that it's incorporated in the same line, the trailing space is better:+1: